mono/mini/mini-arm64.h

```diff
@@ -19,6 +19,13 @@
 
 #define MONO_MAX_IREGS 32
 #define MONO_MAX_FREGS 32
+#define MONO_MAX_XREGS 0
+#define MONO_ARCH_CALLEE_SAVED_XREGS 0
+#define MONO_ARCH_CALLEE_XREGS 0
+
+#if !defined(DISABLE_SIMD) && defined(ENABLE_NETCORE) && defined(ENABLE_LLVM)
+#define MONO_ARCH_SIMD_INTRINSICS 1
```

Comment:
This should not be guarded by ENABLE_LLVM. Instead we should guard by `COMPILE_LLVM (cfg)` for `supported = ...` in `emit_sys_numerics_vector` (with special case for x64).